backend/pokemons/services.py

```diff
@@ -33,11 +33,11 @@ def get_all_pokemons():
     if response.status_code != 200:
         raise Exception("Error fetching pokemon list")
 
-    has_response_next = response.json()["next"]
+    has_response_next = bool(response.json()["next"])
```

Comment:
So this was wrong before, right? You figure it out when testing manually?

Reply:
great, that's why I suggested testing manually, always important to do it at least once when dealing with external apis